### ISSUE TITLE
Word 'Petersilie' is parsed/displayed incorrectly

### ISSUE THREAD

**Issue by radomirbosak** [radomirbosak/duden-down#7]
command:
```
$ python3 dudendown.py Petersilie
```
output:
```
Petersilie
==========
Substantiv, feminin

 0. B

 1. e

 2. d

 3. e

 4. u

 5. t
...
204. r

205. z

206. e

207. l
```

**Comment by radomirbosak** [comment]
Fixed in [version 0.3](https://github.com/radomirbosak/duden-down/releases/tag/v0.3)

**Comment by radomirbosak** [comment]
Fixed in branch [develop](https://github.com/radomirbosak/duden-down/tree/develop).
